src/lib/utils/hooks/lens/profile/useGetLensProfiles.ts

```diff
@@ -32,6 +36,81 @@ export default function useGetLensProfiles(
   );
 }
 
+export function useLensProfilesPerSellerIds(
+  props: {
+    sellers: {
+      authTokenType: number;
+      authTokenId: string;
+      id: string;
+    }[];
+  },
+  options: {
+    enabled?: boolean;
+  }
+) {
+  const { config } = useConfigContext();
+  const lensApiLink = config.lens.apiLink || "";
+  const { enabled } = options;
+  const { sellerIdPerLensToken, lensProfileIds } = props.sellers
+    .filter((seller) => seller.authTokenType === AuthTokenType.LENS)
+    .reduce(
+      (
+        {
+          sellerIdPerLensToken: _sellerIdPerLensToken,
+          lensProfileIds: _lensProfileIds
+        },
+        seller
+      ) => {
+        const sellerId = seller.id;
+        const tokenId = getLensTokenIdHex(seller.authTokenId);
+        if (!_sellerIdPerLensToken.has(tokenId)) {
+          _sellerIdPerLensToken.set(tokenId, sellerId);
+          _lensProfileIds.push(tokenId);
+        }
+        return {
+          sellerIdPerLensToken: _sellerIdPerLensToken,
+          lensProfileIds: _lensProfileIds
+        };
+      },
+      {
+        sellerIdPerLensToken: new Map<string, string>(),
+        lensProfileIds: [] as string[]
```

Comment:
we dont need this lensProfileIds, it can be extracted from the keys of sellerIdPerLensToken